### ISSUE TITLE
CDN Version

### ISSUE THREAD

**Issue by thecodecafe** [catamphetamine/react-phone-number-input#156]
Hi, is it possible to have this package available for CDN, some of us use React via CDN and nor through imports... You're the real mvp though, this package looks very impressive. 👍

**Comment by catamphetamine** [comment]
Hi.
Well, there's the `bundle` folder with files in it.
I can be hosted on a CDN and included on a page.
There are several projects providing CDN capabilities for NPM packages like unpkg.com or jsdelivr.com
Choose any approach you want.

**Comment by catamphetamine** [comment]
Oh, actually, `bundle` is not part of the NPM package.
I'll add it.

**Comment by thecodecafe** [comment]
Okay, on the wait for that.

**Comment by catamphetamine** [comment]
Released `react-phone-number-input-js@1.1.10`.

An example of CDN:

```html
<script src="https://unpkg.com/libphonenumber-js@1.2.15/bundle/libphonenumber-js.min.js"></script>
<script src="https://unpkg.com/react-phone-number-input@1.1.10/bundle/react-phone-number-input.min.js"></script>
```

**Comment by thecodecafe** [comment]
Alright!!!!
You're the best.

**Comment by thecodecafe** [comment]
What about CSS, anything on that?

**Comment by thecodecafe** [comment]
Okay, got them, downloaded them manually.

**Comment by catamphetamine** [comment]
It's a bit more complicated, I'll explain soon.
It's about autoprefixing.

**Comment by thecodecafe** [comment]
Okay, I already have a gulp task, that does that to my scss, so for me I just added it to my project file.

One thing though, after including the cdn to my project I get. `PhoneInput is not defined` maybe some issues with bundling?

**Comment by catamphetamine** [comment]
Try `const PhoneInput = window['react-phone-number-input']`

**Comment by catamphetamine** [comment]
```html
<!-- `libphonenumber-js` (is used internally by `react-phone-number-input`) -->
<script src="https://unpkg.com/libphonenumber-js@1.2.15/bundle/libphonenumber-js.min.js"></script>

<!-- `react-phone-number-input` -->
<script src="https://unpkg.com/react-phone-number-input@1.1.11/bundle/react-phone-number-input.min.js"></script>

<!-- Don't add this style if you're using "native" country select. -->
<link rel="stylesheet" href="https://unpkg.com/react-phone-number-input@1.1.11/bundle/rrui.css"/>

<!-- Styles for the component. ->
<link rel="stylesheet" href="https://unpkg.com/react-phone-number-input@1.1.11/bundle/style.css"/>

<script>
  var PhoneInput = window['react-phone-number-input']
</script>
```

**Comment by thecodecafe** [comment]
So I tried loading it like so `const PhoneInput = window['react-phone-number-input']` and using as show in your example in the documentation, but that yielded an error.

`Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: object.`

Should I create a new issue for this or, should it just be here?

**Comment by catamphetamine** [comment]
Use the new code:


```html
<!-- `libphonenumber-js` (is used internally by `react-phone-number-input`). -->
<script src="https://unpkg.com/libphonenumber-js@1.2.15/bundle/libphonenumber-js.min.js"></script>

<!-- Either `react-phone-number-input` with "native" country `<select/>`. -->
<script src="https://unpkg.com/react-phone-number-input@1.1.12/bundle/react-phone-number-input-native.js"></script>

<!-- Or `react-phone-number-input` with `react-responsive-ui` `<Select/>`. -->
<script src="https://unpkg.com/react-phone-number-input@1.1.12/bundle/react-phone-number-input-react-responsive-ui.js"></script>

<!-- Don't add this style if you're using "native" country `<select/>`. -->
<link rel="stylesheet" href="https://unpkg.com/react-phone-number-input@1.1.12/bundle/rrui.css"/>

<!-- Styles for the component. -->
<link rel="stylesheet" href="https://unpkg.com/react-phone-number-input@1.1.12/bundle/style.css"/>

<script>
  var PhoneInput = window['react-phone-number-input']
</script>
```

`.min` got removed and files got renamed.

**Comment by catamphetamine** [comment]
FYI: I released version `2.1` which now only supports React >= 16.3
